.github/workflows/benchmark.yml

```diff
@@ -24,16 +24,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Git Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           submodules: recursive
       - name: Rust Toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
-          profile: minimal
           # nightly can be very volatile--pin this to a version we know works well...
           toolchain: nightly-2022-08-11
```

Comment:
nit: just noticed that the `benchmark.yml` toolchain (`nightly-2022-08-11`) is different than the `ci_build_test.yml` toolchain. Perhaps these should both be the same?